spec/criteria_spec.cr

```diff
@@ -44,6 +44,24 @@ describe LuckyRecord::Criteria do
       age.lte("30").to_sql.should eq ["SELECT #{QueryMe::COLUMNS} FROM users WHERE age <= $1", "30"]
     end
   end
+
+  describe "not" do
+    describe "without chained criteria" do
```

Comment:
As you're defining conditions for `not` to be run under (with or without chained criteria) I'd use `context` blocks here. Makes it a bit easier to scan back through previous tests, and they are used in other parts of the framework in this way https://github.com/luckyframework/record/blob/master/spec/form_spec.cr